account_mail_templates/__manifest__.py

```diff
@@ -1,24 +1,22 @@
 # Author: Andrius Laukavičius. Copyright: JSC Focusate.
 # See LICENSE file for full copyright and licensing details.
 {
-    'name': "Invoices Custom QWeb Email Templates",
+    'name': "Invoicing Custom QWeb Email Templates",
     'version': '15.0.1.1.0',
-    'summary': 'Invoices jinja2 templates rewritten to QWeb templates',
-    'license': 'AGPL-3',
+    'summary': 'Invoicing templates using QWeb inheritance',
+    'license': 'LGPL-3',
     'author': "Focusate",
     'website': "http://www.focusate.eu",
-    'category': 'Accounting',
+    'category': 'Accounting/Reporting',
     'depends': [
         # odoo
         'account',
-        # oca-social
```

Comment:
Is submodule `oca-social` still necessary for this repo?

Reply:
We will see later

Reply:
As I see, only `sale_mail_templates` and `account_mail_templates` were using `oca-social` dependencies, so probably, could remove submodule or add a TODO somewhere.